fr-French/Localizable.strings

```diff
@@ -10,42 +10,42 @@
 "create" = "Créer";
 "created" = "Créé";
 "delete" = "Supprimer";
-"done" = "Fini";
+"done" = "Terminé";
 "modify" = "Modifier";
 "retry" = "Réessayer";
 "save" = "Sauvegarder";
 // Left menu
 "alias" = "alias";
 "aliases" = "alias";
-"mailbox" = "mailbox (boîte mail)";
-"mailboxes" = "mailbox (boîtes mail)";
+"mailbox" = "boîte mails";
+"mailboxes" = "boîtes mails";
 "settings" = "Réglages";
-"about" = "A propos";
-"rate_us" = "Voter";
+"about" = "À propos";
+"rate_us" = "Notez l'application";
 "sign_out" = "Se déconnecter";
 "sign_out_confirmation" = "Vous allez vous déconnecter";
-"yes_sign_me_out" = "Oui";
+"yes_sign_me_out" = "Oui, déconnectez-moi";
 // Alias
 "forward" = "transfert";
 "forwards" = "transferts";
 "block" = "bloqué";
 "blocks" = "bloqués";
```

Comment:
Singular for "Forward" and "Block" is never used, even if the counter is set to 0 or 1 (it's ok for reply/replies)